x-pack/plugins/cases/common/constants/index.ts

```diff
@@ -164,6 +164,36 @@ export const DELETE_CASES_CAPABILITY = 'delete_cases' as const;
 export const PUSH_CASES_CAPABILITY = 'push_cases' as const;
 export const CASES_CONNECTORS_CAPABILITY = 'cases_connectors' as const;
 
+/**
+ * Cases UI Constants
+ */
+
+// We just need the default ids(field) and isChecked.
+// The name comes from the cases columns configuration.
+export const DEFAULT_CASES_TABLE_COLUMNS: CasesColumnSelection[] = [
```

Comment:
I wanted the default selection to be `CasesColumnSelection` but in practice we really just need the `key/field/id` and the `isChecked` because this gets merged with the configuration in `x-pack/plugins/cases/public/components/all_cases/use_cases_columns_selection.tsx`.

Reply:
Maybe this configuration can be the source of truth about the naming as it is static and cannot change dynamically.

Reply:
But it is not complete, does not include fields like alerts and comments e.g.